common-crafting.lic

```diff
@@ -78,6 +78,16 @@ module DRCC
     end
   end
 
+  def find_loom_room(hometown, override = nil)
+    if override
+      DRCT.walk_to(override)
+    else
+      loomrooms = get_data('crafting')['tailoring'][hometown]['loom-rooms']
+      idle_room = get_data('crafting')['tailoring'][hometown]['idle-room']
+      DRCT.find_sorted_empty_room(loomrooms, idle_room)
```

Comment:
Disclaimer, I'm not familiar with this script. If there isn't an idle room configured, should it fall back to your safe room?

Reply:
@BinuDR @Dartellum - you two have contributed to this script, what do you think?

Reply:
We can add an idle room then. I will add the room right outside the weave rooms.

Reply:
Updated.